views/home.hbs

```diff
@@ -1,78 +1,119 @@
-        <img alt="placeholder" class="thumbnail" src="https://placehold.it/450x210">
-      </div>
-      <div class="cell small-6">
-        <h3>Use unique, strong passwords on every site</h3>
-        <p>It's great</p>
-      </div>
+  <div class="cell"></div>
 
-      <div class="cell small-6">
-        <h3>Use a Password Manager</h3>
-        <p>It's great</p>
-      </div>
-      <div class="cell small-6">
-        <img alt="placeholder" class="thumbnail" src="https://placehold.it/450x210">
-      </div>
+  <div class="cell text-center">
+    <h3>Quick Tips</h3>
+  </div>
 
-    <div class="reveal" id="subscribe-modal" data-reveal>
-      <h3>Subscribe to Breach Alerts</h3>
-      <p>Explanation of what the user will get if they subscribe to breach alerts.</p>
-      <a id="subscribe-fxa-btn" class="button sign-in-btn">Subscribe with Firefox Accounts</a>
-      <div class="grid-x">
-        <div class="cell small-5"><hr></div>
-        <div class="cell small-2">or</div>
-        <div class="cell small-5"><hr></div>
-      </div>
-      <form action="/subscribe" method="post">
-        <input type="email" name="email">
-        <button type="submit" class="button" disabled>Subscribe with Email Account</button>
+  <div class="cell"></div>
+
+  <div class="cell small-1"></div>
+
+  <div class="cell small-5">
+    <h3>Use 2FA</h3>
+    <br>
+    <p>It's great</p>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec volutpat hendrerit !nibus. Donec eu auctor eros. Donec venenatis, eros id suscipit scelerisque, leo orci porta felis, vitae rutrum ipsum odio eu lectus.
+    </p>
+  </div>
+  <div class="cell small-5">
+    <img alt="placeholder" class="thumbnail" src="https://placehold.it/550x310">
+  </div>
+
+  <div class="cell small-1"></div>
+
+  <div class="cell small-1"></div>
+
+  <div class="cell small-5">
+    <img alt="placeholder" class="thumbnail" src="https://placehold.it/550x310">
+  </div>
+  <div class="cell small-5">
+    <h3>Use unique, strong passwords</h3>
+    <br>
+    <p>It's great</p>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec volutpat hendrerit !nibus. Donec eu auctor eros. Donec venenatis, eros id suscipit scelerisque, leo orci porta felis, vitae rutrum ipsum odio eu lectus.
+    </p>
+  </div>
+
+  <div class="cell small-1"></div>
+
+  <div class="cell small-1"></div>
+
+  <div class="cell small-5">
+    <h3>Use a Password Manager</h3>
+    <br>
+    <p>It's great</p>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec volutpat hendrerit !nibus. Donec eu auctor eros. Donec venenatis, eros id suscipit scelerisque, leo orci porta felis, vitae rutrum ipsum odio eu lectus.
+    </p>
+  </div>
+  <div class="cell small-5 padding">
+    <img alt="placeholder" class="thumbnail" src="https://placehold.it/550x310">
+  </div>
+
+  <div class="cell small-1"></div>
+
+
+<div class="reveal grid-x" id="subscribe-modal" data-reveal>
```

Comment:
We need to move this to `default.hbs` too, otherwise the 'Subscribe' button on `/scan` doesn't work.